data/postgresql/postgresql.go

```diff
@@ -1,133 +1,12 @@
 package postgresql
 
 import (
-	"database/sql"
-	"fmt"
-	"log"
-	"time"
-
-	"github.com/haydenwoodhead/burner.kiwi/data"
-	"github.com/haydenwoodhead/burner.kiwi/metrics"
-	"github.com/jmoiron/sqlx"
+	"github.com/haydenwoodhead/burner.kiwi/data/sqldb"
 
 	_ "github.com/lib/pq" // import lib pq here rather than main
 )
 
 // PostgreSQL implements the database interface for postgres
 type PostgreSQL struct {
-	*sqlx.DB
-}
-
-// GetPostgreSQLDB returns a new postgres db or panics
-func GetPostgreSQLDB(dbURL string) *PostgreSQL {
```

Comment:
I think this function needs to come back and wrap `sqldb.GetDatabase`. So that we can actually use `lib/pq` and the sql driver gets registered.